arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/AccountSnapshot.java

```diff
@@ -245,6 +245,15 @@ public AccountSnapshot raiseNonceByOne() {
     return this;
   }
 
+  public AccountSnapshot setDeploymentNumber(Hub hub) {
+    return this.setDeploymentNumber(hub.transients.conflation().deploymentInfo());
+  }
+
+  public AccountSnapshot setDeploymentNumber(DeploymentInfo deploymentInfo) {
+    this.deploymentNumber(deploymentInfo.deploymentNumber(address));
+    return this;
+  }
+
```

Comment:
This is important and reflects a subtle mistake. When we "undo" a row we want to
- copy over the deployment status (criss-cross style)
- get the current deployment number

The `setDeploymentInfo` method overwrites the deployment status. As such it's unsuitable for **undoing** operations.